descriptors/ModuleDescriptor-template.json

```diff
@@ -1033,6 +1044,11 @@
       "displayName" : "fund-update-logs-item delete",
       "description" : "Delete a fund update log"
     },
+    {
+      "permissionName" : "finance-storage.job-number.get",
+      "displayName" : "job-number get",
+      "description" : "Get job number for fund update logs"
```

Comment:
lets not couple to update logs only, we can use this API for other subsequnece jobs also, so can rename desciption

Reply:
removed, thanks